source/core/src/main/com/csse3200/game/entities/factories/LadderFactory.java

```diff
@@ -0,0 +1,56 @@
+package com.csse3200.game.entities.factories;
+
+import com.badlogic.gdx.physics.box2d.BodyDef.BodyType;
+import com.csse3200.game.components.LadderComponent;
+import com.csse3200.game.entities.Entity;
+import com.csse3200.game.physics.PhysicsLayer;
+import com.csse3200.game.physics.components.ColliderComponent;
+import com.csse3200.game.physics.components.PhysicsComponent;
+import com.csse3200.game.rendering.TextureRenderComponent;
+/**
+ * Factory to create Platform entities.
+ *
+ * <p>Each Platform entity type should have a creation method that returns a corresponding entity.
+ */
+public class LadderFactory {
+
+    private LadderFactory() {
+        throw new IllegalStateException("Instantiating static util class");
+    }
+
+    /**
+     * Creates a static ladder entity.
+     * @return entity
+     */
+    public static Entity createStaticLadder() {
+        Entity ladder =
+                new Entity()
+                        .addComponent(new TextureRenderComponent("images/ladder.png"))
```

Comment:
if you change this line then the ladder will always render under the player btw

```suggestion
                        .addComponent(new TextureRenderComponent("images/ladder.png").setLayer(0)
```